recipes-python/pyqtchart/python-pyqtchart_5.13.1.bb

```diff
@@ -0,0 +1,9 @@
+require python-pyqtchart.inc
+
+inherit pythonnative python-dir
+
+DEPENDS += "sip sip-native python python-pyqt5"
+
+RDEPENDS_${PN} += "python-core python-sip python-pyqt5"
+
+PNBLACKLIST[python-pyqt5chart] = "Sip fails in do_configure sip: QOpenGLFramebufferObject is undefined"
```

Comment:
This doesn't do anything, because of the typo in "python-pyqt5chart" and the issue is from python-pyqt5 I guess not from python-pyqtchart.

Reply:
You are referring to which typo?
I have never built against python2, thus this recipe was not tested. It was inspired by python-pyqt5 recipes.

Reply:
python-pyqt5chart != PN

Reply:
I have just fixed the typo.

Reply:
It's not just the typo, whole PNBLACKLIST shouldn't be here. I've already removed it when merging to zeus-next and also fixed python-pyqt5 to build without opengl in DISTRO_FEATURES.

Reply:
Perfect. Thanks.